### ISSUE TITLE
[Suggestion] Use embedded codepen to show example codes in vuepress

### ISSUE THREAD

**Issue by tychenjiajun** [matsp/material-components-vue#242]
**Is your feature request related to a problem? Please describe.**
The vuepress doc now only shows examples but no codes. Users can only get example code from the vuepress source code.

**Describe the solution you'd like**
Implement a customized component `<m-codepen>` in vuepress with an embedded codepen in it. Place `<m-codepen>` in the docs to show the example codes and provide a playground for users.

**Additional context**
[Vuedoc](https://gitlab.com/vuedoc/parser) may also be helpful in writing docs.

**Comment by matsp** [comment]
@tychenjiajun 

You are right. Example code and rendered output side by side would be great.
Let's collect ideas in the issue to evaluate what solutions are possible - maybe also by looking at other projects.

**Comment by tychenjiajun** [comment]
@matsp Personally, I'm in favour of [how Buefy documents their components](https://buefy.org/documentation/field).

**Comment by matsp** [comment]
@tychenjiajun Yeah thats a nice example of documenation!
